Ginger/GingerCoreNET/SourceControl/SourceControlIntegration.cs

```diff
@@ -398,31 +399,25 @@ public static void DownloadSolution(string SolutionFolder)
                 getProjectResult = SourceControlIntegration.CreateConfigFile(mSourceControl);
                 if (getProjectResult != true)
                 {
-                    return;
+                    return false;
                 }
 
                 if (sol.ExistInLocaly == true)
                 {
                     mSourceControl.RepositoryRootFolder = sol.LocalFolder;
-
-
-                    RepositoryItemHelper.RepositoryItemFactory.GetLatest(sol.LocalFolder, mSourceControl);
+                    return RepositoryItemHelper.RepositoryItemFactory.GetLatest(sol.LocalFolder, mSourceControl);
 
                 }
                 else
                 {
-                    getProjectResult = SourceControlIntegration.GetProject(mSourceControl, sol.LocalFolder, ProjectURI);
+                    return getProjectResult = SourceControlIntegration.GetProject(mSourceControl, sol.LocalFolder, ProjectURI);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'getProjectResult'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)